best-practices.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     'block-scoped-var': 'error',
     complexity: ['error', 14],
     'consistent-return': 'error',
+    curly: ['error', 'all'],
```

Comment:
I'd actually disable this one

Reply:
Isn’t it a best practice to always put curly braces around the body? 🤔

You can make less mistakes, since you can’t think that the next line is also conditional, especially when you’re not using semicolons.

```js
if (condition)
  func1()
  func2()
```

vs

```js
if (condition) {
  func1()
}
  func2()
```

Reply:
If we can lint against that then that's fine. But I want to be able to do this:

```js
if (error) throw error
```

Specifically on the same line. If it's multiple lines, then I want the curly braces.

Reply:
I don't think it's possible to only do that for `throw error` tbh.

But we would still have the same problem as before.
If somebody sets a `console.log` before the `throw error`, the error won't be thrown conditionally anymore.
Enforcing the curly braces makes the code less error-prone.

Reply:
But if your wanted to insert a log you'd need to make it multiple lines (warning) or add a semicolon (warning) so I think we're covered.

Reply:
`eslint-config-prettier` suggests against using `multi-line`  and `multi-or-nest`.

https://github.com/prettier/eslint-config-prettier#curly

Reply:
I don't think I agree with their recommendation. Here's what I want:

```javascript
// fine:
if (error) throw error

if (error) {
  throw error
}

// not fine:
if (error) console.log(error); throw error

if (error)
  throw error

// not fine, but prettier fixes so I don't care:
if (error) { throw error }

if (error) { console.log(error); throw error }
```

If we can't make ESLint ok with that, then let's just disable the rule.